src/tests/integration/mapping.test.ts

```diff
@@ -162,4 +162,48 @@ describe('/api', () => {
     const mappingData = await getMapping.json()
     expect(Object.keys(mappingData).length).toEqual(0)
   })
+
+  it('checks same subdomain can be used for different domains', async () => {
+    const subDomain = `testing${uuidv4()}`
+    const domain = 'VinDiesel'
+    const port = '3522'
+    const postResponse = await mappingAdapter('/', 'POST', {
+      domain,
+      subDomain,
+      port
+    })
+    expect(postResponse.status).toEqual(200)
+    const secondDomain = 'PaulWalker'
+    const nextPort = '3523'
+    const secondResponse = await mappingAdapter('/', 'POST', {
+      secondDomain,
+      subDomain,
+      nextPort
+    })
+    expect(secondResponse.status).toEqual(200)
+
+    const postMapping = await postResponse.json()
+    const deleteResponse = await mappingAdapter(
+      `/delete/${postMapping.id}`,
+      'DELETE'
+    )
+    expect(deleteResponse.status).toEqual(200)
+    const secondPostMapping = await secondResponse.json()
+    const secondDeleteResponse = await mappingAdapter(
+      `/delete/${secondPostMapping.id}`,
+      'DELETE'
+    )
+    expect(secondDeleteResponse.status).toEqual(200)
+    const getMapping = await mappingAdapter(`/${postMapping.id}`, 'GET')
+    expect(getMapping.status).toEqual(200)
+    const mappingData = await getMapping.json()
+    const getSecondMapping = await mappingAdapter(
+      `/${secondPostMapping.id}`,
+      'GET'
+    )
+    expect(getSecondMapping.status).toEqual(200)
+    const secondMappingData = await getSecondMapping.json()
+    expect(Object.keys(mappingData).length).toEqual(0)
+    expect(Object.keys(secondMappingData).length).toEqual(0)
```

Comment:
this test is confusing, because it is supposed to test... `check same subdomain can be used for different domains`
tests should be focused on exactly that. I think the most straightforward test to do is:

1. create 1 domain (no need to check response, because other tests took care of domains can be created)
2. create 2nd domain (different domain, same subdomain), check response (should be okay)
3. get all the mappings, expect 1, 2 to be in there
4. delete the mappings